src/audio/Audio.js

```diff
@@ -78,13 +78,20 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
+		if ( when === undefined || when === 0 ) when = this.context.currentTime;
```

Comment:
To properly schedule the stop of the audio when a duration is passed, as we calculate `when + duration` further below.

0 is the default value for `when` in `AudioBufferSourceNode.start()` and so I think it's best to support passing 0 for `when` in `Audio.play()` as well. If 0 is passed, `when` needs to be initialized to the current time for the calculation of the stop time to be correct.